source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/styles/community.applications.css

```diff
@@ -0,0 +1,2392 @@
+    overflow: auto;
+    height: 27rem;
+    width: 36rem;
+    padding-left: .5rem;
+    padding-right: .5rem;
+    background-color: #DDDADF;
+  }
+  
+  .ca_templatesDisplay {
+    display: flex;
+    flex-wrap: wrap;
+  }
+  
+  .ca_template_icon {
+    color: #606E7F;
+    width: 37rem;
+    float: left;
+    display: inline-block;
+    background-color: #C7C5CB;
+    margin: 0px 0px 0px 0px;
+    height: 15rem;
+    padding-top: 1rem;
+  }
+  
+  .ca_twitter::before {
+    content: "\f099";
+    font-family: fontAwesome;
+  }
+  
+  .ca_webpage::before {
+    content: "\f0ac";
+    font-family: fontAwesome;
+  }
+  
+  .changelogMessage {
+    font-size: 1rem;
+    line-height: 1rem;
+    margin-top: 1rem;
+  }
+
+  .card_readmore {
+    color: var(--unraid-66-color);
+  }
+  
+  #Category {
+    font-size: 2rem;
+    margin-bottom: 0.5rem;
+  }
+  
+  .changelogTitle {
+    font-size: 2rem;
+    line-height: 2rem;
+    margin-top: 2rem;
+    font-weight: normal;
+  }
+  
+  .changelog {
+    font-size: 1.2rem;
+    line-height: 1.4rem;
+    margin-top: 1.5rem;
+  }
+  
+  .chartContainer {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+  }
+  
+  .chartMenu {
+    padding-left: 2rem;
+    cursor: pointer;
+  }
+  
+  .chartMenu:hover {
+    color: var(--unraid-66-color);
+  }
+  
+  .chartMenu.selectedMenu {
+    color: var(--unraid-66-color);
+    font-weight: bold;
+  }
+  
+  .charts {
+    font-size: 1.5rem;
+  }
+  
+  .chartTitle {
+    margin-top: 1.5rem;
+    font-size: 2.5rem;
+  }
+  
+  .checkForUpdates {
+    width: 100%;
+    display: none;
+    padding-bottom: 20px;
+  }
+  
+  ul.context {
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+  }
+  
+  a.context {
+    text-decoration: none;
+    color: currentColor;
+    margin: 5px;
+  }
+  
+  li.context {
+    margin-top: 0.5rem;
+    margin-bottom: 0.5rem;
+    font-size: 1.5rem;
+  }
+  
+  li.context:hover {
+    color: var(--unraid-66-color);
+  }
+  
+  .credits {
+    overflow: scroll;
+    overflow-x: hidden;
+    overflow-y: auto;
+    margin: auto;
+    width: 600px;
+    padding-bottom: 50px;
+  }
+  
+  li.debugging {
+    cursor: pointer;
+  }
+  
+  .dimScreenShot {
+    overflow: unset !important;
+  }
+  
+  .disabledIcon {
+    color: #040404;
+    font-size: 2.5rem;
+  }
+  
+  i.displayIcon {
+    font-size: 5.5rem;
+    color: #626868;
+    padding-top: 0.25rem;
+  }
+  
+  img.licence {
+    max-height: 2rem;
+    max-width: 10rem;
+  }
+  
+  img.displayIcon {
+    height: 6.4rem;
+    width: 6.4rem;
+    border-radius: 1rem 1rem 1rem 1rem;
+  }
+  
+  #cookieWarning {
+    display: none;
+  }
+  
+  .docker::after {
+    font-family: 'Unraid';
+    content: '\e90b';
+    font-size: 2.5rem;
+  }
+  
+  .dockerCardBackground {
+    height: 18.5rem;
+    cursor: default !important;
+  }
+  
+  .dockerHubStar {
+    font-size: 1rem;
+  }
+  
+  .dockerHubStar::before {
+    content: "\e95a";
+    font-family: UnRaid;
```

Comment:
Make sure we're using the right case here. First time seeing the R capitalized in the font-family rule.